src/main/resources/assets/carpet/lang/zh_tw.json

```diff
@@ -159,8 +159,8 @@
   "rule.shulkerSpawningInEndCities.name": "可重生界伏蚌",
   "rule.shulkerSpawningInEndCities.desc": "界伏蚌將會在終界城市中重生",
 
-  "rule.silverFishDropGravel.name": "蠹魚鑽出放塊掉落礫石",
-  "rule.silverFishDropGravel.desc": "蠹魚鑽出放塊時會掉落礫石",
+  "rule.silverFishDropGravel.name": "蠹魚掉落礫石",
+  "rule.silverFishDropGravel.desc": "蠹魚死亡時會掉落礫石",
```

Comment:
the same as https://github.com/gnembon/fabric-carpet/pull/410#discussion_r468369398, the old translation is correct, don't change